libcxxabi/src/demangle/ItaniumDemangle.h

```diff
@@ -5424,6 +5435,9 @@ Node *AbstractManglingParser<Derived, Alloc>::parseEncoding() {
         return nullptr;
       Names.push_back(Ty);
     } while (!IsEndOfEncoding() && look() != 'Q');
+    // Ignore the explicit 'this' parameter.
+    if (NameInfo.HasExplicitObjectParameter)
```

Comment:
Any reason to avoid printing out the `this` parameter? I think it could be useful to include it, especially if a user is overloading using it.

Reply:
No particular reason apart from implementation simplicity. Thinking about it some more though, we would be losing the ref-qualifiers on the functions here because in the explicit-object-paramter case that information is encoded into the parameter itself, which we discard.

Should'nt be too difficult to amend this patch to explicitly name the parameter. Probably with a `this` prefix.

Reply:
Had a stab at this in the latest commit